tests/suites/test_suite_psa_crypto_driver_wrappers.data

```diff
@@ -39,3 +39,87 @@ generate_key:PSA_ERROR_NOT_SUPPORTED:"":PSA_SUCCESS
 
 generate_key through transparent driver: error
 generate_key:PSA_ERROR_GENERIC_ERROR:"":PSA_ERROR_GENERIC_ERROR
+
```

Comment:
Please include https://github.com/gilles-peskine-arm/mbedtls/tree/pr_3644-fix_psa_crypto_test_description_copypasta in the pull request. I did this to validate that other than fixing the copypasta (thanks for that), these test cases are copied from existing ones so I don't need to review the test data.

Reply:
Cherry-picked your commit.